### PR TITLE
HSC-57: Add eip config

### PR DESCRIPTION
**Pull request by chinzou**
https://mekomsolutions.atlassian.net/browse/HSC-57

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,6 +36,7 @@
     <!-- OpenMRS modules -->
     <fhir2Version>1.4.0-SNAPSHOT</fhir2Version>
 
+    <eipRoutesOdoo>1.0-SNAPSHOT</eipRoutesOdoo>
```
Comment (rbuisson) [inline pom.xml]: `eipRoutesOdooVersion`

File: eip_config/config/application.properties [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+# *********************** General EIP sender properties ****************************************************************
+#
+
+# Maximum size of each batch of source records to be read by debezium from the binlog files, maps to the
+## debezium MySQL connector property named max.batch.size
+#https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-property-max-batch-size
+debezium.reader.maxBatchSize=2048
+
+#Enabled Oauth when set to true
+oauth.enabled=true
```
Comment (rbuisson) [inline eip_config/config/application.properties]: I don't think we should have this set to `true`

File: eip_config/config/senaite-config/application.properties [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# *********************** General EIP sender properties ****************************************************************
```
Comment (rbuisson) [inline eip_config/config/senaite-config/application.properties]: Is this file needed at all?

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: eip_config/config/senaite-config/application.properties (hunk 3 above)
Comment (chinzou) [reply eip_config/config/senaite-config/application.properties]: This file is not needed at all.

File: eip_config/config/application.properties [hunk 4]
```diff
@@ -14,7 +14,7 @@ openmrs.db.name=${OPENMRS_DB_NAME}
 debezium.reader.maxBatchSize=2048
 
 #Enabled Oauth when set to true
-oauth.enabled=true
+oauth.enabled=false
 
 #The client Id of the account
```
Comment (rbuisson) [inline eip_config/config/application.properties:19]: Should those be removed? or set to empty maybe?

File: eip_config/config/application.properties [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+#Authentication scope, can be multiple values separated by commas
+oauth.client.scope=email
+
+eip.home=/eip-home
+
+# Camel endpoints that need to be notified of DB events
+db-event.destinations=direct:odoo-event-listener,direct:outbound-toLims
+
+# A comma separated list of database tables names to watch for changes
+eip.watchedTables=orders,test_order
```
Comment (rbuisson) [inline eip_config/config/application.properties]: Could you make sure to watch the right tables?
As discussed the other day I believe Odoo routes need to watch more than those 2 tables: See https://github.com/mekomsolutions/eip-routes-odoo/blob/87ea7b58561f2fa961cc347eac7f019e0e5c1e3f/src/test/resources/application.properties#L6

File: pom.xml [hunk 6]
```diff
@@ -35,6 +36,8 @@
     <!-- OpenMRS modules -->
     <fhir2Version>1.4.0-SNAPSHOT</fhir2Version>
 
+    <eipRoutesOdooVersion>1.0-SNAPSHOT</eipRoutesOdooVersion>
```
Comment (rbuisson) [inline pom.xml]: Let's use `1.0.0-SNAPSHOT`

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Pull request merged**